### ISSUE TITLE
Watermelon_ID Not Syncing to Backend DB

### ISSUE THREAD

**Issue by doobe01** [nathanheffley/laravel-watermelon#1]
I have implemented your sync adapter and doing some tests with my react-native app. I cannot get the watermelon_id to populate in my MySQL database. Maybe I am not understanding how this works. I have a sync button in my app that successfully pulls the data from my backend database to WatermelonDB. But if I change anything on the backend and try to sync it does not update the data on the app. Am I missing something? Thanks much!

**Comment by doobe01** [comment]
Error logs when I change something on the backend and try to sync:

[Diagnostic error: [Sync] Server wants client to update record journals#198, but it doesn't exist locally. This could be a serious bug. Will create record instead. If this was intentional, please check the flag sendCreatedAsUpdated in https://nozbe.github.io/WatermelonDB/Advanced/Sync.html#additional-synchronize-flags]

**Comment by doobe01** [comment]
It does end up creating a new database entry on the app with the changed data. So this definitely has something to do with it not having the same ID in both locations. Any ideas? Sorry for the multiple messages.

**Comment by doobe01** [comment]
I figured it out. 😃

**Comment by nathanheffley** [comment]
@doobe01 would you mind writing a comment explaining what you did so that anyone who finds this issue in the future knows how to fix it?

I'm glad to know someone is making use of this package ☺️

**Comment by doobe01** [comment]
Sure. No problem.

I had to generate the watermelon_id in my controller when creating new entries in my database. Once this was added everything started syncing correctly.

` 'watermelon_id' => bin2hex(random_bytes(8)),`

It might worth linking your demos in the readme. Once I was able to find your demo I was able to look through it and figure out my issue. 

Also, thanks much for your work on this. Made this much easier on my end not having to create the sync adapter in Laravel. Think it would be a bit above my head with my current Laravel experience.
